src/plugins/flame-chart-plugin.ts

```diff
@@ -97,7 +97,7 @@ export default class FlameChartPlugin extends UIPlugin {
     }
 
     handleSelect(region) {
-        const selectedRegion = region ? this.findNodeInCluster(region) : null;
+        const selectedRegion = this.findNodeInCluster(region);
```

Comment:
Maybe something unexpected gets into the function call `this.findNodeInCluster(region)` and an exception will occur.

Reply:
ah, makes sense

Reply:
I have rechecked, and findNodeInCluster method already has the same check for truthiness like this one
so it is already guarded

Reply:
Good to hear.) Types are starting to help.
So we can save these changes.